<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupTabPrepare</class>
 <widget class="QWidget" name="SetupTabPrepare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>114</y>
     <width>128</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Rotate:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxFlipVertical">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>48</y>
     <width>128</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Vertical Flip</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRotate">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>128</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxInverse">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>76</y>
     <width>128</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Inverse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>128</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxConversionType">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>23</y>
     <width>128</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBarEdge">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>89</y>
     <width>128</width>
     <height>13</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMonochromeType">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>56</y>
     <width>128</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxFlipHorizontal">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>128</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Horizontal Flip</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
